libs/features/mas/mas/src/mas.min.js

```diff
@@ -0,0 +1,21 @@
+import { init } from '@adobe/mas-commerce';
+
+const { searchParams } = new URL(import.meta.url);
+const locale = searchParams.get('locale') ?? 'US_en';
+const lang = searchParams.get('lang') ?? 'en'; // to be used in the future
+const isStage = searchParams.get('env') === 'stage';
+
+const envName = isStage ? 'stage' : 'prod';
+const commerceEnv = isStage ? 'STAGE' : 'PROD';
```

Comment:
you can also use `envName.toUpperCase()` since the values are the same

Reply:
@narcis-radu I'll do this change in a future PR, because  the code has changed further as part of acom hackathon and I would like to avoid unnecessary merges.